lib/templates/profile/service.html

```diff
@@ -1,3 +1,3 @@
 <a href="#nice-profile" class="menu-profile">
-    <span class="profile-avatar"></span>
+    <span class="profile-avatar" aria-lable="Profile avatar"></span>
```

Comment:
Also, a title gets added to the a tag. We should move the aria-label to the a tag and label is as "Show user menu for X" where X is the name of the logged in user